egs/wsj/s5/steps/nnet3/get_egs.sh

```diff
@@ -55,6 +55,8 @@ online_ivector_dir=  # can be used if we are including speaker information as iV
 cmvn_opts=  # can be used for specifying CMVN options, if feature type is not lda (if lda,
             # it doesn't make sense to use different options than were used as input to the
             # LDA transform).  This is used to turn off CMVN in the online-nnet experiments.
+left_shift_window=false  # If true then left shift window for the last example to accomodate
```

Comment:
this description is confusing. You could write something like
" If false, the last example extracted from an utterance would be padded with the final frame to get frames_per_eg frames.
If true, the starting frame-index of the last example would be shifted to the left to get frames_per_eg frames"